daemon/job/build_jobs.go

```diff
@@ -12,9 +12,17 @@ import (
 )
 
 func JobsFromConfig(c *config.Config) ([]Job, error) {
+	return jobsFromConfigImpl(c, false)
+}
+
+func ParseJobsFromConfig(c *config.Config) ([]Job, error) {
+	return jobsFromConfigImpl(c, true)
+}
+
+func jobsFromConfigImpl(c *config.Config, parseOnly bool) ([]Job, error) {
```

Comment:
Instead of an impl method that takes a `bool`, I suggest we just add a `flags config.ParseFlags` argument to `JobsFromConfig`.
In package `config`, define a `type ParseFlags uint` that is a bitmask.
One bit in that bitmask should be `ParseFlagsNoCertCheck`.